packages/selenium/test/RunPartial/UseLegacyTests.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using OpenQA.Selenium;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Selenium.Test.RunPartial
+{
+    public class UseLegacyTests : TestBase
+    {
+        [Test]
+        [TestCase("Chrome")]
+        [TestCase("Firefox")]
+        public void ShouldRunLegacyAxe(string browser)
+        {
+            InitDriver(browser);
+            GoToFixture("index.html");
+
+#pragma warning disable CS0618
+            var results = new AxeBuilder(WebDriver, CustomSource($"{axeSource}{axeRunPartialThrows}"))
+                .UseLegacyMode(true)
+                .Analyze();
+#pragma warning restore CS0618
+
+            Assert.IsNotNull(results);
+            Assert.IsTrue(results.Passes.Any());
+        }
+
+        [Test]
+        [TestCase("Chrome")]
+        [TestCase("Firefox")]
+        public void ShouldNotSetAOWhenRunPartialAndNotLegacy(string browser)
+        {
+            InitDriver(browser);
+            GoToFixture("index.html");
+            new AxeBuilder(WebDriver).Analyze();
+            var allowedOrigins = AllowedOrigins();
+            Assert.That(allowedOrigins, Is.EqualTo(ExpectedAllowedOrigins(browser)));
+        }
+
+        [Test]
+        [TestCase("Chrome")]
+        [TestCase("Firefox")]
+        public void ShouldNotSetAOWhenRunPartialAndLegacy(string browser)
+        {
+            InitDriver(browser);
+            GoToFixture("index.html");
+#pragma warning disable CS0618
+            new AxeBuilder(WebDriver)
+                .UseLegacyMode(true)
+                .Analyze();
+#pragma warning restore CS0618
+            var allowedOrigins = AllowedOrigins();
+            Assert.That(allowedOrigins, Is.EqualTo(ExpectedAllowedOrigins(browser)));
+        }
+
+        [Test]
+        [TestCase("Chrome")]
+        [TestCase("Firefox")]
+        public void ShouldNotSetAOWhenLegacySourceAndLegacyMode(string browser)
+        {
+            InitDriver(browser);
+            GoToFixture("index.html");
+#pragma warning disable CS0618
+            new AxeBuilder(WebDriver, CustomSource($"{axeSource}{axeForceLegacy}"))
```

Comment:
This should use the legacy version of axe and not the one that forces it to be legacy.